src/main.rs

```diff
@@ -77,7 +77,7 @@ fn app() -> Result {
     cmds.add("?tags help", tags::help);
 
     // crates.io
-    cmds.add("?crate", crates::help);
+    cmds.add("?crate help", crates::help);
```

Comment:
Unfortunately since the `query...` command segment captures _everything_ the `help` in `?crate help` is never matched.  You have to disambiguate them which is why I suggested `?crate search query...`

Reply:
the way the state machine works, commands defined earlier have higher priority - it tries to match `help` and then falls back to capturing everything. if i were to switch these two lines `crate help.` would never match like you said

Reply:
Awesome, im fine with this.